src/execution/collectFields.ts

```diff
@@ -265,6 +271,7 @@ function collectFieldsImpl(
             fragmentVariableSignatures,
             variableValues,
             fragmentVariableValues,
+            false,
```

Comment:
I think this has to be `maskSuggestions` instead of false because this is where we get the fragment variable values, so it's not just about the built ins.